privx_api/vault.py

```diff
@@ -48,6 +61,27 @@ def get_secrets(
         )
         return PrivXAPIResponse(response_status, HTTPStatus.OK, data)
 
+    def get_user_secrets(
+        self,
+        userid: str,
```

Comment:
Use  "user_id" instead of "userid". Same comment for any occurrences of "userid"